packages/fxa-settings/src/components/Banner/interfaces.ts

```diff
@@ -16,7 +17,8 @@ import { NotificationType } from '../../models';
  */
 export type BannerProps = {
   type: NotificationType;
-  content: BannerContentProps;
+  content?: BannerContentProps;
+  fullContent?: ReactNode;
```

Comment:
Perhaps this content option should be a new option in the `BannerContentProps`, unless `fullContent` could be included _in addition to_ the other types of content?

Reply:
Heh I did that at first, but I wasn't a big fan of needing to add `full?: never` to the other options - this felt a bit more flexible to me. It can't be done in addition to the other pieces of content (like keeping `localizedHeader`) unless we want to add some condition also on class names, it just made the component feel a bit harder to understand. I could take another look at it though.

Reply:
In that case, I'd nitpick and suggest adding the `fullContent` below the rest of the content in the component (in case it's ever an addition to the heading, I don't think the "fullContent" would be placed above the rest of the content strings)